src/GitHub.App/Services/TeamExplorerContext.cs

```diff
@@ -128,33 +122,20 @@ async Task RefreshAsync()
                         log.Debug("ActiveRepository changed to {CloneUrl} @ {Path}", repo?.CloneUrl, newRepositoryPath);
                         ActiveRepository = repo;
                     }
-                    else if (newBranchName != branchName)
-                    {
-                        log.Debug("Fire StatusChanged event when BranchName changes for ActiveRepository");
-                        StatusChanged?.Invoke(this, EventArgs.Empty);
-                    }
-                    else if (newHeadSha != headSha)
```

Comment:
So StatusChanged is no longer firing when HEAD changes? Is this because it wasn't working before anyway? (i.e. it was firing before the actual change)